### ISSUE TITLE
[FR] Accessing GDX files from Julia

### ISSUE THREAD

**Issue by yordiak** [GAMS-dev/gams.jl#4]
Are you planning to release a tool, similar to `gdxrrw` in `R`? I know that you can access `R` packages by using `RCall.jl`. However, I would prefer a more direct data exchange between `GAMS` and `Julia`.

**Comment by renkekuhlmann** [comment]
Yes, this is on our list (incl. accessing our "expert-level" interfaces, e.g. gdx, gmo etc. from Julia). I hope to get this done for GAMS 34 or 35 (GAMS 33 will be released shortly). Is there any particular format you would like us to consider to write the gdx data to (like data frames)?

**Comment by yordiak** [comment]
I believe that `data frames` is perfect, even though it's a little bit restrictive to someone who wants a more general format. Thus, I would suggest to keep `gdx data` as generic as possible. If someone wants to convert it to `data frame` then she/he can simply do it by using `DataFrames.jl` package

**Comment by MartinBonde** [comment]
Any news on this?

**Comment by renkekuhlmann** [comment]
Sorry, no news at this point.

**Comment by renkekuhlmann** [comment]
User may want to have a look at Martin Bonde's contribution: https://github.com/MartinBonde/GamsGDX.jl
